.gitignore

```diff
@@ -12,6 +13,9 @@ project.lock.json
 # NUnit
 TestResult.xml
 
+# NCruch
+_NCrunch*
```

Comment:
http://www.ncrunch.net/ - ncrunch is a unit test runner it adds stuff to the directory that shouldn't be checked in. 

I'll fix the title